lib/Github/Api/GraphQL.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Github\Api;
+
+/**
+ * GraphQL API.
+ *
+ * @link   http://developer.github.com/v3/markdown/
+ * @author Miguel Piedrafita <soy@miguelpiedrafita.com>
+ */
+class GraphQL extends AbstractApi
+{
+    /**
+     * @param string $query
+     *
+     * @return array
+     */
+    public function graphql($query)
```

Comment:
Do we want the signature to be: 
`$github->graphql()->graphql($query)`

Compare to: 
`$github->graphql()->execute($query)`